cli/pyproject.toml

```diff
@@ -6,7 +6,7 @@ authors = [
 ]
 description = "PIXL command line interface"
 readme = "README.md"
-requires-python = ">=3.10"
+requires-python = "<3.11"
```

Comment:
oh why do we have this change?

Reply:
Because current setup doesn't work for 3.12 (not tested with 3.11) :(
See also [#195](https://github.com/UCLH-Foundry/PIXL/issues/195)